test/scenarios/presubmits/el96-src@rpm-install.sh

```diff
@@ -112,5 +112,7 @@ scenario_run_tests() {
         --variable "SOURCE_REPO_URL:${source_repo_url}" \
         --variable "TARGET_VERSION:${target_version}" \
         --variable "PREVIOUS_MINOR_VERSION:${PREVIOUS_MINOR_VERSION}" \
-        suites/rpm/install-and-upgrade-successful.robot
+        suites/rpm/install.robot \
+        suites/rpm/remove.robot \
+        suites/rpm/upgrade-successful.robot
```

Comment:
Why are we listing tests one-by-one?

Reply:
Because we need to run them in that exact order: install, remove and upgrade from previous version.